src/components/AudioPlayer/Player.js

```diff
@@ -0,0 +1,428 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import colors from '../../theme/colors'
+import { mUp } from '../../theme/mediaQueries'
+import { css, merge } from 'glamor'
+import { timeFormat } from '../../lib/timeFormat'
+import { breakoutStyles } from '../Center'
+import { InlineSpinner } from '../Spinner'
+import Volume from '../VideoPlayer/Icons/Volume'
+import Play from 'react-icons/lib/md/play-arrow'
+import Pause from 'react-icons/lib/md/pause'
+
+const ZINDEX_AUDIOPLAYER_ICONS = 6
+const ZINDEX_AUDIOPLAYER_SCRUB = 3
+const ZINDEX_AUDIOPLAYER_PROGRESS = 2
+const ZINDEX_AUDIOPLAYER_BUFFER = 1
+const ZINDEX_AUDIOPLAYER_TOTAL = 0
+const PROGRESS_HEIGHT = 4
+
+const hoursDurationFormat = timeFormat('%-H:%M:%S')
+const minutesDurationFormat = timeFormat('%-M:%S')
+
+const barStyle = {
+  position: 'absolute',
+  height: PROGRESS_HEIGHT,
+  top: -PROGRESS_HEIGHT,
+  left: 0,
+  right: 0
+}
+
+const styles = {
+  wrapper: css({
+    position: 'relative',
+    height: '40px',
+    lineHeight: 0
+  }),
+  audio: css({
+    height: 0,
+    width: '100%',
+    '::-webkit-media-controls-panel': {
+      display: 'none !important'
+    },
+    '::--webkit-media-controls-play-button': {
+      display: 'none !important'
+    },
+    '::-webkit-media-controls-start-playback-button': {
+      display: 'none !important'
+    }
+  }),
+  controls: css({
+    position: 'absolute',
+    top: 10,
+    left: 0,
+    right: 0,
+    cursor: 'pointer',
+    height: '25px'
+  }),
+  play: css({
+    position: 'absolute',
+    top: '50%',
+    left: 0,
+    marginTop: -15,
+    textAlign: 'center'
+  }),
+  mute: css({
+    position: 'absolute',
+    top: '50%',
+    left: 40,
+    marginTop: -12,
+    textAlign: 'center'
+  }),
+  progress: css({
+    position: 'absolute',
+    zIndex: ZINDEX_AUDIOPLAYER_PROGRESS,
+    backgroundColor: colors.primary,
+    top: -PROGRESS_HEIGHT,
+    left: 0,
+    height: PROGRESS_HEIGHT
+  }),
+  time: css({
+    position: 'absolute',
+    zIndex: ZINDEX_AUDIOPLAYER_ICONS,
+    right: 10,
+    cursor: 'pointer',
+    fontSize: '16px',
+    lineHeight: '25px',
+    height: '30px',
+    color: colors.text,
+    [mUp]: {
+      fontSize: '19px'
+    }
+  }),
+  scrub: css({
+    ...barStyle,
+    zIndex: ZINDEX_AUDIOPLAYER_SCRUB,
```

Comment:
give it at least 25px to scrub without being a precision machine, with something like this:
```
height: 20,
marginTop: -((20 + PROGRESS_HEIGHT) / 2)
paddingTop: 20 / 2 - PROGRESS_HEIGHT / 2
```